src/output.jl

```diff
@@ -147,13 +147,14 @@ saveproblem(out::Output) = saveproblem(out.prob, out.path)
 """
     savediagnostic(diag, diagname, filename)
 
-Save diagnostics in `diag` to `filename`, labeled by `diagname`.
+Save diagnostic in `diag` to `filename` under name `diagname`. Only the computed diagnostic
```

Comment:
This is clear but could maybe change to 

Save the diagnostic `diag`...

after is good.

Reply:
I changed everything to `diagnostic` instead of `diag`.